km/km_vcpu_run.c

```diff
@@ -37,6 +37,7 @@
 
 int vcpu_dump = 0;
 int km_collect_hc_stats = 0;
+int kill_unimpl_hcall = 0;
```

Comment:
move to machine.
Slowly move to no gobal variables.

Reply:
Make it an environment variable as well, easier to set in various environments.

Reply:
I'm going to create an environment variable this one, but I'm going to hold off putting it into `machine`.

Our current config stuff is a bit of a mess. I'd like to see a more generalized `config` service that could deal with command line options and environment variables. I see it presenting a common way to inquire if an option is enabled and/or an optional argument is present.

I'd also like to review what we have now and see if it all still makes sense.